grafana/overlays/moc/smaug/blackbox-exporter/service-monitor.yaml

```diff
@@ -21,6 +21,72 @@ spec:
           - 'https://jupyterhub-opf-jupyterhub.apps.smaug.na.operate-first.cloud/'
       path: /probe
       targetPort: 9115
+    - interval: 30s
+      metricRelabelings:
+        - sourceLabels:
+            - __address__
+          targetLabel: __param_target
+        - sourceLabels:
+            - __param_target
+          targetLabel: instance
+        - replacement: 'https://www.operate-first.cloud/'
+          targetLabel: target
+      params:
+        module:
+          - http_2xx
+        target:
+          - 'https://www.operate-first.cloud/'
+      path: /probe
+    - interval: 30s
+      metricRelabelings:
+        - sourceLabels:
+            - __address__
+          targetLabel: __param_target
+        - sourceLabels:
+            - __param_target
+          targetLabel: instance
+        - replacement: 'https://thanos-query-frontend-opf-observatorium.apps.smaug.na.operate-first.cloud/'
+          targetLabel: target
+      params:
+        module:
+          - http_2xx
+        target:
+          - 'https://thanos-query-frontend-opf-observatorium.apps.smaug.na.operate-first.cloud/'
+      path: /probe
+      targetPort: 10902
+    - interval: 30s
+      metricRelabelings:
+        - sourceLabels:
+            - __address__
+          targetLabel: __param_target
+        - sourceLabels:
+            - __param_target
+          targetLabel: instance
+        - replacement: 'https://trino.operate-first.cloud/'
+          targetLabel: target
+      params:
+        module:
+          - http_2xx
+        target:
+          - 'https://trino.operate-first.cloud/'
+      path: /probe
+    - interval: 30s
+      metricRelabelings:
+        - sourceLabels:
+            - __address__
+          targetLabel: __param_target
+        - sourceLabels:
+            - __param_target
+          targetLabel: instance
+        - replacement: 'http://cloudbeaver-opf-trino.apps.smaug.na.operate-first.cloud/'
+          targetLabel: target
+      params:
+        module:
+          - http_2xx
+        target:
+          - 'http://cloudbeaver-opf-trino.apps.smaug.na.operate-first.cloud/'
+      path: /probe
+      targetPort: 8978
   namespaceSelector:
```

Comment:
Also need [`spec.selector`](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#servicemonitorspec). It can be empty:
```yaml
spec:
    selector: {}
```